{
  "schema_version": "1.6.3",
  "id": "USN-4668-1",
  "summary": "python-apt vulnerability",
  "details": "Kevin Backhouse discovered that python-apt incorrectly handled resources. A\nlocal attacker could possibly use this issue to cause python-apt to consume\nresources, leading to a denial of service.\n",
  "aliases": [],
  "related": [
    "CVE-2020-27351"
  ],
  "published": "2020-12-09T16:36:45.504991Z",
  "modified": "2020-12-09T16:36:45.504991Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:16.04:LTS",
        "name": "python-apt",
        "purl": "pkg:deb/ubuntu/python-apt@1.1.0~beta1ubuntu0.16.04.10?arch=src?distro=xenial"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.1.0~beta1ubuntu0.16.04.10"
            }
          ]
        }
      ],
      "versions": [
        "1.0.1build1",
        "1.0.1ubuntu2",
        "1.1.0~beta1",
        "1.1.0~beta1build1",
        "1.1.0~beta1ubuntu0.16.04.1",
        "1.1.0~beta1ubuntu0.16.04.2",
        "1.1.0~beta1ubuntu0.16.04.4",
        "1.1.0~beta1ubuntu0.16.04.5",
        "1.1.0~beta1ubuntu0.16.04.7",
        "1.1.0~beta1ubuntu0.16.04.8",
        "1.1.0~beta1ubuntu0.16.04.9"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-apt": "1.1.0~beta1ubuntu0.16.04.10",
            "python-apt-common": "1.1.0~beta1ubuntu0.16.04.10",
            "python-apt-dbg": "1.1.0~beta1ubuntu0.16.04.10",
            "python-apt-dev": "1.1.0~beta1ubuntu0.16.04.10",
            "python-apt-doc": "1.1.0~beta1ubuntu0.16.04.10",
            "python3-apt": "1.1.0~beta1ubuntu0.16.04.10",
            "python3-apt-dbg": "1.1.0~beta1ubuntu0.16.04.10"
          }
        ],
        "availability": "No subscription required"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:18.04:LTS",
        "name": "python-apt",
        "purl": "pkg:deb/ubuntu/python-apt@1.6.5ubuntu0.4?arch=src?distro=bionic"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.6.5ubuntu0.4"
            }
          ]
        }
      ],
      "versions": [
        "1.4.0~beta3build2",
        "1.4.0~beta3ubuntu1",
        "1.6.0~rc2ubuntu1",
        "1.6.0~rc2ubuntu2",
        "1.6.0",
        "1.6.1",
        "1.6.2",
        "1.6.3",
        "1.6.3ubuntu1",
        "1.6.4",
        "1.6.5ubuntu0.1",
        "1.6.5ubuntu0.2",
        "1.6.5ubuntu0.3"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-apt": "1.6.5ubuntu0.4",
            "python-apt-common": "1.6.5ubuntu0.4",
            "python-apt-dbg": "1.6.5ubuntu0.4",
            "python-apt-dev": "1.6.5ubuntu0.4",
            "python-apt-doc": "1.6.5ubuntu0.4",
            "python3-apt": "1.6.5ubuntu0.4",
            "python3-apt-dbg": "1.6.5ubuntu0.4"
          }
        ],
        "availability": "No subscription required"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:20.04:LTS",
        "name": "python-apt",
        "purl": "pkg:deb/ubuntu/python-apt@2.0.0ubuntu0.20.04.2?arch=src?distro=focal"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2.0.0ubuntu0.20.04.2"
            }
          ]
        }
      ],
      "versions": [
        "1.9.0ubuntu1",
        "1.9.0ubuntu3",
        "1.9.0ubuntu5",
        "1.9.3ubuntu2",
        "1.9.5",
        "1.9.5build1",
        "1.9.7",
        "1.9.8",
        "1.9.10",
        "2.0.0",
        "2.0.0ubuntu0.20.04.1"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-apt": "2.0.0ubuntu0.20.04.2",
            "python-apt-common": "2.0.0ubuntu0.20.04.2",
            "python-apt-dbg": "2.0.0ubuntu0.20.04.2",
            "python-apt-dev": "2.0.0ubuntu0.20.04.2",
            "python-apt-doc": "2.0.0ubuntu0.20.04.2",
            "python3-apt": "2.0.0ubuntu0.20.04.2",
            "python3-apt-dbg": "2.0.0ubuntu0.20.04.2"
          }
        ],
        "availability": "No subscription required"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://ubuntu.com/security/notices/USN-4668-1"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2020-27351"
    }
  ]
}